cli/src/main/java/com/devonfw/tools/ide/url/model/UrlMetadata.java

```diff
@@ -87,7 +87,7 @@ private List<VersionIdentifier> computeSortedVersions(String tool, String editio
       try {
         urlVersion.getMatchingUrls(sys.getOs(), sys.getArchitecture());
         list.add(versionIdentifier);
-      } catch (IllegalStateException e) {
+      } catch (CliException e) {
```

Comment:
we have completed the last release and opened 2025.08.001 as SNAPSHOT version. Could you now add the story to the changelog?
p.s.: I know this review comment is misplaced but GitHub does not over another way to add a review CR not related to a code-line that was changed and when I write it only in the review comment summary it often gets forgotten.
But since I am annotation this line: I already checked that this is the only single exception situation so the change is safe.
However, just as a thought we could also sub-class and throw + catch a more specific variant of `CliException` for this scenario.